ervinu/web content/style.css

```diff
@@ -0,0 +1,105 @@
+body {
+    background-color: rgb(211, 207, 207);
+    margin: 10px 5px;
+    font-family: sans-serif;
+}
+
+#top-container {
+    background-color: #ffffff;
+    margin: 5px;
+}
+
+#title h2, #title a {
+    display: inline-block;
+    margin: 5px 10px;
+}
+
+a {
+text-decoration: none;
+}
+
+.carousel {
+    background-color: rgb(255, 255, 255);
+    height: 330px;
+    width: 99%;
+    position: relative;
+    display: flex;
+}
+
+.carousel p {
+    margin: 4px 0px;
+}
+
+.carousel img {
+    background-color: rgb(236, 236, 236);
+    height: 70%;
+    width: 98%;
+    margin: 0px 10px;
+}
+
+
+.deal-discount p:first-child {
+    background-color: rgb(255, 0, 0);
+    color: rgb(255, 255, 255);
+    display: inline-block;
+    margin-left: 10px;
+}
+
+.deal-discount p:last-child {
+    color: red;
+    display: inline-block;
+}
+
+.euro-sign {
+    font-size: 0.5em;
+    position: relative;
+    top: -0.25em;
+    right: -0.25em;
+    top: -5px;
+}
+
+.cents {
+    font-size: 0.5em;
+    position: relative;
+    top: -5px;
+    right: -1px;
+}
+
+.was {
+    font-size: 0.8em;
+    opacity: 0.5;
+}
+
+.old-price {
+    font-size: 0.8em;
+    opacity: 0.5;
+    text-decoration: line-through;
+}
+
+#bottom-container {
+    margin-top: 20px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.galery {
+    background-color: rgb(255, 255, 255);
+    margin: 5px;
+    width: 340px;
+    overflow: hidden;
+}
+
+.galery img {
+    margin: 0px 15px;
+    width: 90%;
```

Comment:
why 90%?

Reply:
Otherwise the images inside their divs, are not positioned as they should be

Reply:
90% not going to work on different screen sizes. Try to let them align by default. Use flexbox as much as you can